src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3420,9 +3420,34 @@
     public static final int ELLIPSIS = GeneratedJavaTokenTypes.ELLIPSIS;
 
     /**
-     * The {@code &} symbol when used in a generic upper or lower bounds constrain
-     * e.g&#46; <code>Comparable&lt;T extends Serializable &amp; CharSequence&gt;</code>.
-     * @noinspection HtmlTagCanBeJavadocTag
+     * The {@code &} symbol when used to extend a generic upper or lower bounds constrain
+     * or a type cast expression with an additional interface.
+     *
+     * <p>Generic type bounds extension:
+     * {@code Comparable<T extends Serializable & CharSequence>}</p>
+     * <pre>
+     * +--TYPE_UPPER_BOUNDS (extends)
+     *     |--IDENT (Serializable)
+     *     |--TYPE_EXTENSION_AND (&amp;)
```

Comment:
It is better to keep Comparable in tree , whole tree of code

Reply:
done